ci/cron/daily-compat.yml

```diff
@@ -173,6 +173,60 @@ jobs:
                                        --project daml
           fi
 
+  - job: update canton community source
+    timeoutInMinutes: 60
+    pool:
+      name: ubuntu_20_04
+      demands: assignment -equals default
+    steps:
+    - checkout: self
+      persistCredentials: true
+    - template: ../bash-lib.yml
+      parameters:
+        var_name: bash_lib
+    - bash: |
+        set -euo pipefail
+
+        eval "$(./dev-env/bin/dade-assist)"
+        source $(bash_lib)
+
+        git fetch
+        git checkout origin/main
+
+        tmp=$(mktemp -d)
+        trap "rm -rf ${tmp}" EXIT
+
+        git clone git@github.com:DACH-NY/canton.git --depth 1  $tmp
+
+        for path in community daml-common-staging; do
```

Comment:
I would like to keep this a bit generic.
`path` can be a directory or a file and can be arbitrary nested in the directory structure of canton.